main.sh

```diff
@@ -4,17 +4,17 @@ CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 [[ -z "$TMUX_FZF_ORDER" ]] && TMUX_FZF_ORDER="copy-mode|session|window|pane|command|keybinding|clipboard|process"
 source "$CURRENT_DIR/scripts/.envs"
 
+items="$(echo $TMUX_FZF_ORDER | tr '|' '\n')"
+
 # remove copy-mode from options if we aren't in copy-mode
-if [[ "$TMUX_FZF_ORDER" == *"copy-mode"* ]] && [ "$(tmux display-message -p '#{pane_in_mode}')" -eq 0 ]; then
-    TMUX_FZF_ORDER="$(echo $TMUX_FZF_ORDER | sed -E 's/\|?copy-mode\|?//')"
+if [ "$(tmux display-message -p '#{pane_in_mode}')" -eq 0 ]; then
+    items_origin="$(echo "${items}" | sed '/copy-mode/d')"
```

Comment:
IMO the functionality didn't change here. Doesn't `sed -E 's/\|?copy-mode\|?//'` work on your machine?